Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/README.md

```diff
@@ -88,215 +53,205 @@ RBAC Permissions Required - org.alerts.dismiss: EXECUTE
 | CarbonBlackEEDR.Alert.State | String | The alert state. | 
 
 
-##### Command Example
-```!cb-eedr-alert-workflow-update alert_id=A28C720DCBCD66333A624893AB1E0FE9 state=open```
+#### Command Example
+``` ```
 
-##### Context Example
-```
-{
-    "CarbonBlackEEDR.Alert": {
-        "Comment": null, 
-        "ChangedBy": "ATL5Y9DR4B", 
-        "AlertID": "A28C720DCBCD66333A624893AB1E0FE9", 
-        "LastUpdateTime": "2020-05-26T13:33:12.890Z", 
-        "State": "OPEN", 
-        "Remediation": null
-    }
-}
-```
+#### Human Readable Output
 
-##### Human Readable Output
-### Successfully updated the alert: "A28C720DCBCD66333A624893AB1E0FE9"
-|changed_by|last_update_time|state|
-|---|---|---|
-| ATL5Y9DR4B | 2020-05-26T13:33:12.890Z | OPEN |
 
 
-### 2. cb-eedr-device-quarantine
----
+### cb-eedr-device-quarantine
+***
 Quarantines a device.
-##### Required Permissions
-RBAC Permissions Required - device.quarantine: EXECUTE
-##### Base Command
+
+
```

Comment:
you have deleted the permissions of this and the following commands, please re add them